.pending/improvements/sdk/4584-Update-bank-Kee

```diff
@@ -0,0 +1,2 @@
+#4584 Update bank Keeper to use expected keeper interface of the AccountKeeper.
+#4584 Move `Account` and `VestingAccount` interface types to `x/auth/expected`
```

Comment:
This should say “exported” instead (afk)

Reply:
whoops, typo! Danke